libraries/botbuilder-lg/tests/lgDiagnostic.test.js

```diff
@@ -187,6 +188,14 @@ describe(`LGExceptionTest`, function() {
         assert.strictEqual(diagnostics[0].message.includes(TemplateErrors.noEndingInMultiline), true);
     });
 
+    it(`TestMultiLineTemplate`, function() {
+        var diagnostics = preloaded.MultiLineTemplate;
+
+        assert.strictEqual(1, diagnostics.length);
```

Comment:
These params are backward; `actual` comes first and `expected` is second.

Reply:
Yes. I fixed some of them. I found there are many places that should change. I think I could file an issue and fix them all at one time besides this PR. Is that ok?

Reply:
Yeah, great idea!